targets/drive/mobile/main.jsx

```diff
@@ -38,78 +43,80 @@ if (__DEVELOPMENT__) {
 }
 
 // Register callback for when the app is launched through cozydrive:// link
-window.handleOpenURL = require('drive/mobile/lib/handleDeepLink').default(hashHistory)
-
-const renderAppWithPersistedState = (persistedState = {}) => {
-  const cozyURL = persistedState.mobile
-    ? persistedState.mobile.settings.serverUrl
-    : ''
-  configureReporter()
-  const client = initClient(cozyURL)
-  const store = configureStore(client, persistedState)
-
-  const clientInfos = store.getState().settings.client
-  if (clientInfos) {
-    client.isRegistered(clientInfos).then(isRegistered => {
-      if (isRegistered) {
-        startReplication(store.dispatch, store.getState) // don't like to pass `store.dispatch` and `store.getState` as parameters, big coupling
-        initBar(client)
-      } else {
-        console.warn('Your device is no more connected to your server')
-        store.dispatch(revokeClient())
-      }
+window.handleOpenURL = require('drive/mobile/lib/handleDeepLink').default(
+  hashHistory
+)
+
+function uploadIntent(intent, store) {
+  if (intent['android.intent.extra.STREAM']) {
```

Comment:
What happens if the condition fails here? Nothing as far as I can tell, but will that happen?

Reply:
I don't know, this code must be refactored and live a bit to catch every intent application can send as Android's intents are free to customize.